ccmlib/node.py

```diff
@@ -150,7 +151,9 @@ def load(path, name, cluster):
             binary_interface = None
             if 'binary' in itf and itf['binary'] is not None:
                 binary_interface = tuple(itf['binary'])
-            node = cluster.create_node(data['name'], data['auto_bootstrap'], tuple(itf['thrift']), tuple(itf['storage']), data[
+            # auto_bootstrap is deprecated
+            auto_bootstrap = True
+            node = cluster.create_node(data['name'], auto_bootstrap, tuple(itf['thrift']), tuple(itf['storage']), data[
```

Comment:
I guess we cannot remove the param from `create_node` because it might break users? (public api)

Reply:
Yeah even the main user is dtest

There are usage across our driver forks as well